README.md

```diff
@@ -68,24 +68,24 @@ in Sass):
 #### `asset-path($relative-asset-path)`
 Returns a string to the asset.
 
-* `asset-path("rails.png")` becomes `"/assets/rails.png"`
```

Comment:
I just did. Do you think I should also update this from my PR https://github.com/rails/rails/pull/21143 for the Rails Asset Pipeline guide? Readers might be confused otherwise.